packages/flame/lib/src/components/mixins/draggable.dart

```diff
@@ -8,18 +8,37 @@ mixin Draggable on Component {
   bool _isDragged = false;
   bool get isDragged => _isDragged;
 
+  /// Override this to handle the start of a drag/pan gesture that is within the
+  /// boundaries (or hitboxes) of the component that this mixin is used on.
```

Comment:
Maybe mention that "within boundaries" means as determined by [containsPoint] method, and remove mention of hitboxes (which may be confusing to someone who doesn't know about hitboxes).

Reply:
Good idea.